contract/src/main/kotlin/com/ritense/valtimo/contract/authentication/NamedUser.kt

```diff
@@ -18,14 +18,34 @@ package com.ritense.valtimo.contract.authentication
 
 data class NamedUser(
     val id: String,
+    val email: String?,
```

Comment:
Please make this the last argument with a default of null, to make it non-breaking (in most cases)

Reply:
Added a constructor